src/xrCore/Memory/XRayAllocator.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
Do you restore the file which was deleted in the previous commit? Or do you made some changes? Anyway, I think we should avoid removing it at all in the previous commit.